### PR TITLE
Directly specify password in the 'password-update' command

### PR DESCRIPTION
**Pull request by lolski**
## Usage and product changes

It is now possible to invoke `password-update` command and specify the password together, instead of having to supply them in the following prompt. This new variant makes it possible to call it non-interactively.

## Implementation
1. Add a new variant to the password-update command: `user password-update <old-password> <new-password>`
2. Update the help text accordingly

### CODE REVIEW

File: TypeDBConsole.java [hunk 1]
```diff
@@ -409,6 +409,17 @@ private boolean runInlineCommandMode(CLIOptions options, List<String> inlineComm
...
+                        REPLCommand.User.PasswordUpdate userPasswordUpdate = command.asUserPasswordUpdate();
+                        boolean passwordUpdateSuccessful = runUserPasswordUpdate(driver,
+                                isCloud,
+                                options.username,
+                                userPasswordUpdate.passwordOld(),
+                                userPasswordUpdate.passwordNew());
+                        if (passwordUpdateSuccessful) {
+                            printer.info("Please login again with your updated password.");
+                            break;
+                        }
```
Comment (krishnangovindraj) [inline TypeDBConsole.java:422]: Do you need an `else { return false; }` ?

File: .factory/automation.yml [hunk 2]
```diff
@@ -44,7 +44,7 @@ build:
     deploy-runner-maven-snapshot:
       filter:
         owner: vaticle
-        branch: [master, development]
+        branch: [master, development, password-update-variant]
```
Comment (krishnangovindraj) [inline .factory/automation.yml:47]: Needs revert before merge, here and CircleCI config.

**Review by krishnangovindraj — CHANGES_REQUESTED** [review]

File: TypeDBConsole.java (hunk 1 above)
Comment (lolski) [reply TypeDBConsole.java]: Good catch! Fixed.

**Review by krishnangovindraj — APPROVED** [review]
lgtm

**Pull request merged**